sendtx/tx_sender.go

```diff
@@ -131,7 +131,7 @@ func (txSnd *TxSender) CreateTxGeneric(
 	receiverAddr string,
 	metadata []byte,
 	outputLovelace uint64,
-	outputNativeToken *cardanowallet.TokenAmount,
+	outputNativeTokens ...*cardanowallet.TokenAmount,
```

Comment:
I think we do not need pointer here. Pointer was just in case native token not provided (empty list or nil is similar)